content/blog/atomic-design-for-spas.md

```diff
@@ -0,0 +1,213 @@
++++
+title = "Atomic Design in Single Page Applications"
+description = "How I use Atomic Design to architect my SPAs in React"
+date = 2018-10-25
++++
+
+This post will outline how I architect visual part of the SPAs I work on. I used
+to put every components in a `components` folder, which obviously doesn't scale and make
+it hard finding a specific component. After that, I grouped at least the top level routing
+components which made things a bit tidier but was still very unsatisfying.
+Looking around for various ways to organise design elements, I found [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/).
+I am not going to dive deep into explaning what is Atomatic Design as the linked article does a very good job
+so I would recommend taking 2 minutes to read it beforehand.
```

Comment:
looks like it'll take more than 2 min to read though

Reply:
it seems long but there are lots of image so it is pretty fast to read